app/models/foreman_azure_rm/azure_rm.rb

```diff
@@ -113,7 +113,7 @@ def new_vm(args = {})
           ifaces << new_interface(iface_attrs)
         end
       end
-      AzureRmCompute.new(azure_vm: raw_vm ,sdk: sdk, resource_group: opts[:resource_group], nics: ifaces)
+      AzureRmCompute.new(azure_vm: raw_vm ,sdk: sdk, resource_group: opts[:resource_group], nics: ifaces, script_command: opts[:script_command], script_uris: opts[:script_uris])
```

Comment:
But, if you see here in `new_vm`, we do pass all the `opts` to `initialize_vm` and in initialize_vm method we do not handle anything related to script_command and script_uris.  Instead I had passed them directly to AzureRmCompute.new call here, how we did for nics.  So, should we still pass these two in `intialize_vm`?